app/views/taxons/_form_fields.html.erb

```diff
@@ -8,7 +8,7 @@
       value: page.taxon.path_slug.presence || page.taxon.path_prefix
     } %>
 
-  <div class="base-path-hint"'>
```

Comment:
Spotting things like this is great, but given this is unrelated, being in a separate commit would be even better.

Reply:
👍